packages/sheets-ui/src/services/editor-bridge.service.ts

```diff
@@ -340,8 +340,9 @@ export class EditorBridgeService extends Disposable implements IEditorBridgeServ
             worksheet.getCell(startRow, startColumn),
             location
         );
+        const cellDefaultStyle = worksheet.getCellDefaultStyle(startRow, startColumn, skeleton.isRowStylePrecedeColumnStyle);
 
-        documentLayoutObject = cell && skeleton.getCellDocumentModelWithFormula(cell);
+        documentLayoutObject = cell && skeleton.getCellDocumentModelWithFormula(cell, cellDefaultStyle);
```

Comment:
Is this the only place that `cellDefaultStyle` should be passed? `getCellDocumentModelWithFormula` are called in other situations such as removing cell hyperlink.